net-libs/daq/daq-2.0.7-r1.ebuild

```diff
@@ -0,0 +1,84 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="Data Acquisition library, for packet I/O"
+HOMEPAGE="https://www.snort.org/"
+SRC_URI="https://www.snort.org/downloads/snort/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~sparc ~x86"
+IUSE="ipv6 +afpacket +dump +pcap nfq ipq static-libs"
+
+PCAP_DEPEND=">=net-libs/libpcap-1.0.0"
+IPT_DEPEND="
+	>=net-firewall/iptables-1.4.10
+	dev-libs/libdnet
+	net-libs/libnetfilter_queue
+
+"
+DEPEND="
+	dump? ( ${PCAP_DEPEND} )
+	ipq? ( ${IPT_DEPEND} )
+	nfq? ( ${IPT_DEPEND} )
+	pcap? ( ${PCAP_DEPEND} )
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-2.0.6-parallel-grammar.patch #673390
+	"${FILESDIR}"/${PN}-2.0.6-static-libs.patch
+	"${FILESDIR}"/${PN}-2.0.7-gcc14-build-fix.patch
+)
+
+src_prepare() {
+	default
+	eautoreconf
+}
+
+src_configure() {
+	# We forced libpcap to 1.x, so we can set this cache var so
+	# cross-compiling doesn't break on us.
+	daq_cv_libpcap_version_1x=yes \
+	econf \
+		$(use_enable afpacket afpacket-module) \
+		$(use_enable dump dump-module) \
+		$(use_enable ipq ipq-module) \
+		$(use_enable ipv6) \
+		$(use_enable nfq nfq-module) \
+		$(use_enable pcap pcap-module) \
+		$(use_enable static-libs static) \
+		--disable-bundled-modules \
+		--disable-ipfw-module \
+		--enable-shared
+}
+
+DOCS=( ChangeLog README )
```

Comment:
`DOCS` could probably be removed, these files should be already installed per default